src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,99 @@
+const expensesService = require('../services/expenses.service');
+const usersService = require('../services/users.service');
+
+let lastId = 0;
+
+function getExpenses(req, res) {
+  const searchParams = new URLSearchParams(req.url.split('?')[1]);
+
+  const userId = searchParams.get('userId');
+  const category = searchParams.get('categories');
+  const from = searchParams.get('from');
+  const to = searchParams.get('to');
+
+  res.send(expensesService.getSortedExpenses(userId, category, from, to));
+}
+
+function getExpense(req, res) {
+  const id = req.params.id;
+  const response = expensesService.getOne(id);
+
+  if (response) {
+    res.send(response);
+
+    return;
+  }
+
+  res.sendStatus(404);
```

Comment:
```suggestion
try {
  const id = req.params.id;
  const response = expensesService.getOne(id);

  if (response) {
    res.send(response);

    return;
  }

  res.sendStatus(404);
  } catch (err) {
  res.sendStatus(500);
  }
```

Reply:
Consider wrapping code in controllers into try-catch blocks and sending the 500 error if something on the server doesn't work